Source/WebKit/Shared/Extensions/_WKWebExtensionUtilities.mm

```diff
@@ -80,38 +81,88 @@ + (BOOL)validateContentsOfDictionary:(NSDictionary<NSString *, id> *)dictionary
     NSSet<NSString *> *optionalKeysSet = [[NSSet alloc] initWithArray:optionalKeys];
 
     __block NSString *errorString;
+
+    auto validateArray = ^BOOL(NSString *key, NSObject *value, NSArray<Class> *validClassesArray) {
+        ASSERT(validClassesArray.count == 1);
+
+        Class expectedElementType = validClassesArray.firstObject;
+        NSArray *arrayValue = dynamic_objc_cast<NSArray>(value);
+        if (!arrayValue) {
+            errorString = [NSString stringWithFormat:@"Expected an array for '%@'.", key];
```

Comment:
Nit: Do we want to put the class we found instead?

Reply:
Yeah that would be good. Will do.